recipes/epc/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "epc" %}
+{% set version = "0.0.5" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a14d2ea74817955a20eb00812e3a4630a132897eb4d976420240f1152c0d7d25" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: 'python setup.py install '
+
+requirements:
+  host:
+    - python
+    - sexpdata >=0.0.3
+  run:
+    - python
+    - sexpdata >=0.0.3
+
+test:
+  imports:
+    - epc
+    - epc.tests
+
+about:
+  home: https://github.com/tkf/python-epc
+  license: GNU General Public License v3 (GPLv3)
+  license_family: GPL3
+  license_file: ''
```

Comment:
GPL needs a license file packaged with the package.

Reply:
What does this mean? I need to copy the license into the recipe?

Reply:
If the license is in the tarball then you can just put the path to the license. Otherwise you'll need to copy the license over. https://conda-forge.org/docs/meta.html#packaging-the-license-manually